relay-general/src/pii/attachments.rs

```diff
@@ -325,7 +328,7 @@ impl<'a> FusedIterator for WStrSegmentIter<'a> {}
 /// longer match.
 struct WStrSegment<'a> {
     /// The raw bytes of this segment.
-    encoded: &'a mut WStr,
+    encoded: &'a mut WStr<LittleEndian>,
```

Comment:
It would be nice to expose typedefs for this as typing out `WStr<LittleEndian>` every time is a little cumbersome.

Reply:
The shorthand is `WStr<LE>` which is also exposed as `utf16string::LE`.  For signatures and struct definitions I choose the longer spelling as this isn't that common around here and makes reading easier when you're not used to it (i.e. myself in a year's time)

Reply:
Makes sense, being explicit here helps for sure. I was thinking of `utf16string::{WStrL, WStrB}` or something similar to avoid the angle brackets (similar to how `io::Result` is a shorthand for `Result<io::Error>`), but in the end it doesn't really matter.